packages/editor/package.json

```diff
@@ -19,9 +19,13 @@
   ],
   "dependencies": {
     "@axonivy/process-editor-protocol": "~13.1.0-next",
+    "@axonivy/process-editor-view": "~13.1.0-next",
+    "@axonivy/ui-components": "~13.1.0-next.451",
```

Comment:
this should be a peer dependency like ui-icons

Reply:
Will change it.